src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from '../Todo/Todo';
+
+export const TodoList = ({
+  todos,
+  removeTodo,
+  toggleSetCompleted,
+  toggleSetAllCompleted,
+}) => (
+  <>
+    <input
+      type="checkbox"
+      id="toggle-all"
```

Comment:
why do you need id here?

Reply:
We have input and label tegs, which connected with attributes id and htmlFor